### ISSUE TITLE
Task watchdog got triggered after hotspot activation by pressing on the Configure button

### ISSUE THREAD

**Issue by TheSomeMan** [ruuvi/ruuvi.gateway_esp.c#403]
```
I (111933) reset_task: [reset_task] The CONFIGURE button has been pressed - start timer for 5 seconds
I (111933) LEDS: [reset_task] leds_indication_on_configure_button_press
I (111939) LEDS: [reset_task] LED: Start blinking, interval: 500 ms, duty cycle: 50%
I (112132) reset_task: [reset_task] The CONFIGURE button has been released
I (112132) reset_task: [reset_task] Disconnect from Ethernet
I (112135) wifi_manager: [wifi_manager] MESSAGE: ORDER_DISCONNECT_ETH
I (112142) ruuvi_gateway: [wifi_manager] callback: on_disconnect_eth_cmd
I (112150) wifi_manager: [wifi_manager] Set STA IP String to: 0.0.0.0
I (112157) ETH: [wifi_manager] Ethernet stop
I (112163) ETH: [sys_evt] Ethernet Stopped
I (112167) reset_task: [reset_task] WiFi AP is not active - stop Ethernet and start WiFi AP
I (112175) ETH: [reset_task] Ethernet stop
W (112180) esp_eth: driver not started yet
E (112185) ETH: [reset_task] ../main/ethernet.c:381 {ethernet_stop}: Ethernet stop failed, err=259 (ESP_ERR_INVALID_STATE)
I (112197) wifi_manager: [reset_task] wifi_manager_start_ap
I (112203) wifi_manager: [wifi_manager] MESSAGE: ORDER_START_AP
I (112210) wifi:mode : sta (94:b9:7e:4e:b2:70) + softAP (94:b9:7e:4e:b2:71)
I (112220) wifi:Total power save buffer number: 16
I (112221) wifi_manager: [sys_evt] WIFI_EVENT_AP_START
I (112229) LEDS: [reset_task] leds_indication_on_hotspot_activation
I (112234) LEDS: [reset_task] LED: Start blinking, interval: 2000 ms, duty cycle: 50%
I (112243) ruuvi_gateway: [reset_task] Start AP timer for 60 seconds
I (112249) reset_task: [reset_task] The CONFIGURE button has been pressed - start timer for 5 seconds
I (112259) LEDS: [reset_task] leds_indication_on_configure_button_press
I (112266) LEDS: [reset_task] LED: Start blinking, interval: 500 ms, duty cycle: 50%
I (112275) reset_task: [reset_task] The CONFIGURE button has been released
I (112282) reset_task: [reset_task] WiFi AP is already active
I (113189) ruuvi_gateway: [main] free heap: 129084
I (123189) ruuvi_gateway: [main] free heap: 129084
I (132511) wifi:new:<1,0>, old:<1,0>, ap:<1,0>, sta:<0,0>, prof:1
I (132512) wifi:station: d6:23:f1:e7:21:9b join, AID=1, bgn, 20
I (132528) wifi_manager: [sys_evt] WIFI_EVENT_AP_STACONNECTED
I (132528) wifi_manager: [wifi_manager] MESSAGE: EVENT_AP_STA_CONNECTED
I (132531) ruuvi_gateway: [wifi_manager] callback: on_ap_sta_connected
I (132538) ruuvi_gateway: [wifi_manager] Stop AP timer
I (132544) dns_server: [wifi_manager] Start DNS-Server
I (132550) os_task: [wifi_manager] Start thread 'dns_server' with priority 4, stack size 3072 bytes
I (132559) dns_server: [dns_server] DNS-Server thread started
I (132567) dns_server: [dns_server] DNS Server listening on 53/udp
I (132573) dns_server: [dns_server] TaskWatchdog: Register current thread
I (132580) dns_server: [dns_server] TaskWatchdog: Start timer
I (133189) ruuvi_gateway: [main] free heap: 123652
I (133850) tcpip_adapter: softAP assign IP to station,IP is: 10.10.0.2
I (133851) wifi_manager: [sys_evt] IP_EVENT_AP_STAIPASSIGNED
I (133853) wifi_manager: [wifi_manager] MESSAGE: EVENT_AP_STA_IP_ASSIGNED
I (133921) ADV_POST_TASK: [adv_post_task] Advertisements in table: 1
I (133922) ADV_POST_TASK: [adv_post_task] i: 0, tag: F4:C6:46:2C:3E:B4, rssi: -30, timestamp: 1635420805:
I (133927) ADV_POST_TASK: [adv_post_task] 0000: 02 01 06 1B FF 99 04 05 14 33 1F 8D C5 5D 00 24 | .........3...].$
I (133938) ADV_POST_TASK: [adv_post_task] 0010: 00 04 03 F0 A9 36 47 C2 FB F4 C6 46 2C 3E B4    | .....6G....F,>.
I (133951) http: [adv_post_task] HTTP POST to URL=https://network.ruuvi.com/record, DATA:
{
        "data": {
                "coordinates":  "",
                "timestamp":    "1635420807",
                "nonce":        "2154584215",
                "gw_mac":       "C8:25:2D:8E:9C:2C",
                "tags": {
                        "F4:C6:46:2C:3E:B4":    {
                                "rssi": -30,
                                "timestamp":    "1635420805",
                                "data": "0201061BFF99040514331F8DC55D0024000403F0A93647C2FBF4C6462C3EB4"
                        }
                }
        }
}
I (134857) tcpip_adapter: softAP assign IP to station,IP is: 10.10.0.2
I (134857) wifi_manager: [sys_evt] IP_EVENT_AP_STAIPASSIGNED
I (134860) wifi_manager: [wifi_manager] MESSAGE: EVENT_AP_STA_IP_ASSIGNED
I (136893) dns_server: [dns_server] Replying to DNS request for gsp85-ssl.ls.apple.com from 10.10.0.2
I (136898) dns_server: [dns_server] Replying to DNS request for gsp85-ssl.ls.apple.com from 10.10.0.2
I (136903) dns_server: [dns_server] Replying to DNS request for api.mixpanel.com from 10.10.0.2
I (136912) dns_server: [dns_server] Replying to DNS request for captive.apple.com from 10.10.0.2
I (136922) dns_server: [dns_server] Replying to DNS request for captive.apple.com from 10.10.0.2
I (136931) dns_server: [dns_server] Replying to DNS request for p18-mailws.icloud.com from 10.10.0.2
I (136941) dns_server: [dns_server] Replying to DNS request for p18-mailws.icloud.com from 10.10.0.2
I (137054) http_server: [http_server] Request from 10.10.0.2 to 10.10.0.1: GET /hotspot-detect.html
I (137055) http_server: [http_server] Respond: 302 Found
I (137078) http_server: [http_server] Request from 10.10.0.2 to 10.10.0.1: GET /
I (137079) http_server: [http_server] http_server_handle_req_get /
I (137082) http_server: [http_server] http_server_cb_on_get /index.html
I (137091) http_server: [http_server] Respond: 200 OK
I (138225) http_server: [http_server] Request from 10.10.0.2 to 10.10.0.1: GET /hotspot-detect.html
I (138226) http_server: [http_server] Respond: 302 Found
I (138248) http_server: [http_server] Request from 10.10.0.2 to 10.10.0.1: GET /
I (138249) http_server: [http_server] http_server_handle_req_get /
I (138253) http_server: [http_server] http_server_cb_on_get /index.html
I (138261) http_server: [http_server] Respond: 200 OK
I (138326) http_server: [http_server] Request from 10.10.0.2 to 10.10.0.1: GET /hotspot-detect.html
I (138327) http_server: [http_server] Respond: 302 Found
I (138335) http_server: [http_server] Request from 10.10.0.2 to 10.10.0.1: GET /jquery-3.5.1.js
I (138341) http_server: [http_server] http_server_handle_req_get /jquery-3.5.1.js
I (138350) http_server: [http_server] http_server_cb_on_get /jquery-3.5.1.js
I (138360) dns_server: [dns_server] Replying to DNS request for safebrowsing.googleapis.com from 10.10.0.2
I (138363) http_server: [http_server] Respond: 200 OK
I (138381) dns_server: [dns_server] Replying to DNS request for safebrowsing.googleapis.com from 10.10.0.2
I (138451) http_server: [http_server] Request from 10.10.0.2 to 10.10.0.1: GET /crypto-js-4.0.0/core.js
I (138452) http_server: [http_server] http_server_handle_req_get /crypto-js-4.0.0/core.js
I (138460) http_server: [http_server] http_server_cb_on_get /crypto-js-4.0.0/core.js
I (138469) http_server: [http_server] Respond: 200 OK
I (138487) http_server: [http_server] Request from 10.10.0.2 to 10.10.0.1: GET /crypto-js-4.0.0/md5.js
I (138490) http_server: [http_server] http_server_handle_req_get /crypto-js-4.0.0/md5.js
I (138495) http_server: [http_server] http_server_cb_on_get /crypto-js-4.0.0/md5.js
I (138505) http_server: [http_server] Respond: 200 OK
I (138525) http_server: [http_server] Request from 10.10.0.2 to 10.10.0.1: GET /crypto-js-4.0.0/sha256.js
I (138526) http_server: [http_server] http_server_handle_req_get /crypto-js-4.0.0/sha256.js
I (138534) http_server: [http_server] http_server_cb_on_get /crypto-js-4.0.0/sha256.js
I (138544) http_server: [http_server] Respond: 200 OK
I (138559) http_server: [http_server] Request from 10.10.0.2 to 10.10.0.1: GET /css/style.css
I (138560) http_server: [http_server] http_server_handle_req_get /css/style.css
I (138568) http_server: [http_server] http_server_cb_on_get /css/style.css
I (138577) http_server: [http_server] Respond: 200 OK
I (138604) http_server: [http_server] Request from 10.10.0.2 to 10.10.0.1: GET /ruuvi.js
I (138605) http_server: [http_server] http_server_handle_req_get /ruuvi.js
I (138610) http_server: [http_server] http_server_cb_on_get /ruuvi.js
I (138618) http_server: [http_server] Respond: 200 OK
I (138641) http_server: [http_server] Request from 10.10.0.2 to 10.10.0.1: GET /
I (138642) http_server: [http_server] http_server_handle_req_get /
I (138646) http_server: [http_server] http_server_cb_on_get /index.html
I (138655) http_server: [http_server] Respond: 200 OK
I (138700) http_server: [http_server] Request from 10.10.0.2 to 10.10.0.1: GET /code.js
I (138701) http_server: [http_server] http_server_handle_req_get /code.js
I (138706) http_server: [http_server] http_server_cb_on_get /code.js
I (138714) http_server: [http_server] Respond: 200 OK
I (138887) http_server: [http_server] Request from 10.10.0.2 to 10.10.0.1: GET /assets/fonts/Montserrat-ExtraBold.woff2
I (138892) http_server: [http_server] http_server_handle_req_get /assets/fonts/Montserrat-ExtraBold.woff2
I (138898) http_server: [http_server] http_server_cb_on_get /assets/fonts/Montserrat-ExtraBold.woff2
I (138910) http_server: [http_server] Respond: 200 OK
E (139102) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
E (139102) task_wdt:  - adv_post_task (CPU 0/1)
E (139102) task_wdt: Tasks currently running:
E (139102) task_wdt: CPU 0: wifi
abort() was called at PC 0x400829a2 on core 0

ELF file SHA256: 2ec0c1d014d9727a

Backtrace: 0x40087a95:0x3ffaffd0 0x40087e15:0x3ffafff0 0x400829a2:0x3ffb0010 0x40082ac5:0x3ffb0040 0x40186cfd:0x3ffb0060 0x40189dad:0x3ffb0370 0x400820b5:0x3ffb03a0 0x400820d9:0x3ffb03d0 0x4014ab75:0x3ffb0420 0x4015f6d6:0x3ffb0490 0x4014cb4a:0x3ffb0500 0x4015224e:0x3ffb0550 0x40152fc1:0x3ffb05a0 0x401508f2:0x3ffb05c0 0x40153c90:0x3ffb05e0 0x4014920a:0x3ffb0600 0x400819d2:0x3ffb0620 0x400d9747:0x3ffb0640 0x400d4dfc:0x3ffb0660 0x400827d2:0x3ffb0680 0x40089aba:0x3ffd31e0 0x40089c1d:0x3ffd3220 0x40084ba1:0x3ffd3240 0x4008d3b6:0x3ffd3260 0x4009179d:0x3ffd32c0 0x4008a7d1:0x3ffd32f0

Rebooting...
ets Jul 29 2019 12:21:46

```

**Comment by markoaamunkajo** [comment]
[Uploading loki.txt…]()
Task watchdog got triggered. Spotted in logs 1.9.0

**Comment by TheSomeMan** [comment]
It's another problem, see #424, fixed in v1.9.1

**Comment by markoaamunkajo** [comment]
Moving to done
